.travis.yml

```diff
@@ -0,0 +1,47 @@
+sudo: false
+language: python
+python: '3.4'
+cache:
+  directories:
+    - ~/.cache/pip
+    - ~/.bundle
+    - ~/.npm
+    - ~/.bower-packages
+env:
+  global:
+    - PIP_WHEEL_DIR=$HOME/.cache/pip/wheels
+    - PIP_FIND_LINKS=file://$HOME/.cache/pip/wheels
+    - CXX=g++-4.8
+    - bower_storage__packages=$HOME/.bower-packages
+  matrix:
+    - TEST_SUITE=api
+    - TEST_SUITE=web
+    - TEST_SUITE=integration
+before_install:
+    - nvm install 5.0.0
+    - nvm use 5.0.0
+    # Start X window server before other tasks so it has time to boot.
+    - export DISPLAY=:99.0;
+    - sh -e /etc/init.d/xvfb start;
+install:
+    - make travis-install-$TEST_SUITE
+script: make test-$TEST_SUITE
+notifications:
+    email: false
+addons:
+    postgresql: "9.3"
+    apt:
+        sources:
+            - ubuntu-toolchain-r-test
+        packages:
+            - fontforge-nox
+            - g++-4.8
+branches:
+  only:
+    - master
+    # Version branches: 1.x, 1.2, 1.2.x, 1.x.x, 1.2.3, 11.22.33, but not 1, x.1
+    - /^\d+(\.(x|\d+)){1,2}$/
+    # Develop and feature branches: develop, developfoo, develop-1, feature-foo,
+    # but not feature, feature-
+    - /^develop/
+    - /^feature-./
```

Comment:
This travis setup has many things you don't need. It would be better to copy one from an epatient lib